pegasus/sites.v3/code.org/public/csjourneys-alt.partial

```diff
@@ -2,6 +2,7 @@
 title: CS Journeys - Explore the many worlds of computer science
 video_player: true
 theme: responsive
+page_banner: csjourneys_header_banner
```

Comment:
Mostly curious -- why is `page_banner` a string instead of a boolean value?  (This string ends up referring to the filename of the partial which might be slightly unexpected.  Wondering if we do that elsewhere?)

Reply:
This makes the page_banner flexible, so other pages could have a page_banner element which would span the full width of the top of the page. It follows the pattern of how we've implemented `nav` on pegasus pages, for example on https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/research.md we have `nav: educate_nav` which is different from https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/about/2020.md.partial where we have `nav: about_nav`